tests/by-util/test_wc.rs

```diff
@@ -464,3 +464,41 @@ fn test_files0_from_with_stdin_try_read_from_stdin() {
         .stderr_contains(MSG)
         .stdout_is("");
 }
+
+#[cfg(unix)]
+#[test]
+fn test_bash() {
+    let wc_path = path_concat!(env!("OUT_DIR"), "..", "..", "..", "coreutils");
```

Comment:
I think `TestScenario` also exposes it with the `bin_path` field. So I think this could be

```rust
let wc_path = scene.bin_path; 
```